### PR TITLE
INT-1752 Use better editor

### PR DESCRIPTION
**Pull request by hbjORbj**
Screenshot 2023-09-06 at 09 51 07

### CODE REVIEW

File: intuita-webview/package.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
 	"dependencies": {
 		"@monaco-editor/react": "^4.5.0",
 		"@tippyjs/react": "^4.2.6",
+		"@tiptap/pm": "^2.1.8",
```
Comment (grzpab) [inline intuita-webview/package.json:8]: how much it enlarges the build size?

File: intuita-webview/src/sourceControl/CreateIssue/tiptap.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.tiptap {
```
Comment (grzpab) [inline intuita-webview/src/sourceControl/CreateIssue/tiptap.css:1]: did you write these CSS?

File: intuita-webview/package.json (hunk 1 above)
Comment (grzpab) [reply intuita-webview/package.json:8]: license: MIT https://www.npmjs.com/package/@tiptap/pm

File: intuita-webview/package.json [hunk 3]
```diff
@@ -5,6 +5,9 @@
...
+		"@tiptap/react": "^2.1.8",
```
Comment (grzpab) [inline intuita-webview/package.json:9]: license: MIT https://www.npmjs.com/package/@tiptap/react

File: intuita-webview/package.json [hunk 4]
```diff
@@ -5,6 +5,9 @@
...
+		"@tiptap/starter-kit": "^2.1.8",
```
Comment (grzpab) [inline intuita-webview/package.json:10]: license: MIT https://www.npmjs.com/package/@tiptap/starter-kit

File: intuita-webview/src/sourceControl/CreateIssue/tiptap.css (hunk 2 above)
Comment (hbjORbj) [reply intuita-webview/src/sourceControl/CreateIssue/tiptap.css:1]: yep

**Pull request merged**